content/community/gsoc/2021/ideas.html

```diff
@@ -359,13 +353,11 @@ <h4><img src="/images/App_Generic_32.png"/>
 </ul>
 
 
-<h4><img src="/images/App_Generic_32.png"/>
-Porting Apple NTFS driver to Haiku
-</h4>
+<h3><img src="/images/App_Generic_32.png"/>Porting Apple NTFS driver to Haiku</h3>
 
 <p>Haiku currently supports NTFS using code from the NTFS-3G project, which is
-distributed under the GPL license. We would prefer to use Apple implementation,
-which is under 3 clause BSD, more suitable for us.</p>
+distributed under the GPL license. We would prefer to use Apple's implementation,
+which is under the 3 clause BSD license, which is a more suitable license for us.</p>
```

Comment:
the double "which is" and repetition of "license" here doesn't make this really better